server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/remoteDelivery/DelaysAndMaxRetryTest.java

```diff
@@ -133,4 +133,11 @@ public void getExpendedDelaysShouldExpandMultipleDelays() throws Exception {
 
         assertThat(testee.getExpendedDelays()).containsExactly(1000L, 2000L, 2000L, 5000L, 5000L);
     }
+    
+    @Test
+    public void getExpendedDelaysShouldExpandMultipleDelaysWithSpaces() throws Exception {
```

Comment:
as you spotted the typo on Expended vs Expanded, maybe a separated commit with this refactoring would be a good idea ?